src/robot_module/src/robot_module/main.py

```diff
@@ -0,0 +1,115 @@
+import rospy
+from .services.movement import Movement
+
+# Service topics
+# subscriber_topics = {
+#    "cameras": lambda: ImageSub("/nautilus/cameras/stream"),
+#    "img_h": lambda: ImageSub("/image/distribute")
+#}
+
+# Available Services
+services = {
+    "movement": lambda node_name: Movement(node_name)
```

Comment:
If I'm understanding the code correctly, the only service we're planning to use acts as a lock around the movement control. Are any additional services being planned for in the future?

Reply:
Getting Camera information is planned for the future.